### ISSUE TITLE
retention policy to retain a snapshot for every n of time

### ISSUE THREAD

**Issue by liabri** [pcal43/fastback#168]
Let me give an example, to keep at least 1 snapshot for every 1 week, regardless if this 1 week was 24 weeks ago (where we would now have 24 snapshots for every one of those 24 weeks). I like to see the history of my world, and with the way snapshots work I don't think space is a particularly big issue.

**Comment by pcal43** [comment]
Hi.  There currently is a `daily` [retention policy](https://pcal43.github.io/fastback/diskspace.html):
```
Daily: Keep the last snapshot from each day, plus all snapshots from the last n days
```

Just so I understand: are you saying you want one that works like that except that it keeps one per week rather than one per day?

**Comment by liabri** [comment]
Well the week was just an example, I would likely set it up to maybe 2-3 weeks, but my point is a configurable period would be nice!

**Comment by pcal43** [comment]
Right.  But I'd prefer to keep things as simple as possible.  A lot of backup tools do daily/weekly/monthly as a standard policy (Grandfather-father-son).


If there were a weekly policy, and space doesn't matter much, would that be close enough? (Weekly vs every 2-3 weeks)

**Comment by liabri** [comment]
I think weekly is just fine, not perfect but I understand the battle of weighing complexity/simplicity and features. If you consider it to be too complex for what it is, keeping it at the typical daily/weekly/monthly policies is just fine I think.
